docs/sidecar-configuration-format.md

```diff
@@ -59,6 +59,13 @@ env:
 volumeMounts:
   - name: some-config
     mountPath: /etc/some-config
+
+# initContainers will be added, no replacement of existing initContainers with the same names will be done
+# this works exactly the same way like adding normal containers does: if you have a conflicting name, the server will return an error
```

Comment:
I suggest breaking `the server will return an error` on a new line

Reply:
Done